src/bot/scripts/card.js

```diff
@@ -728,946 +66,1233 @@ const card = new Command({
   preload: true,
 
   // -card <name>
-  cb: async function (client, message, guild) {
-
-    // deep copy cards so we can replace shit
-    // without worrying about gay object shit
-    const cardsCopy = JSON.parse(JSON.stringify(cards));
-
-    // render frames for the website
-    if (config.env != null &&
-      config.env.toLowerCase() === "dev" &&
-      message.content.split(" ")[1] === "dev-frames") {
-
-      renderFrames(cardsCopy);
-      return;
-    }
-
-    // legacy command support
-
-    // card <name>
-    // card <name> <level>
-    // card <name> l<level>
-    // card <name> u<upgrade>
-
-    let name = null;
-
-    let level = null;
-    let upgrade = null;
-
-    // 5 = card(4) + space(1).
-    const args = message.content.slice(guild.prefix.length + 5);
-    if (args === "") {
+  cb: async function (client, message) {
+    if (Date.now() - new Date(cache.date) > hourInMs) {
```

Comment:
youre literally not using the cache stuff anywhere else in the code, so its just making an extra request every hour and nothing more as it currently is

Reply:
isnt that what its supposed to do? every time the cache refreshes

Reply:
yes, but youre not using the values that are cached anywhere else in your code so its pointless rn

Reply:
done